WickedEngine/wiRenderPath3D.cpp

```diff
@@ -1280,7 +1280,7 @@ namespace wi
 			RenderPassImage rp[] = {
 				RenderPassImage::RenderTarget(
 					&rtMain_render,
-					RenderPassImage::LoadOp::DONTCARE
+					RenderPassImage::LoadOp::CLEAR
```

Comment:
We only want to clear if sky is not rendered, otherwise it is more optimal to not clear.